public/app/features/alerting/unified/mocks/server/handlers.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Contains definitions for all handlers that are required for test rendering of components within Alerting
```

Comment:
We want to talk about handlers/functions that generate handlers outside of the tests for multiple reasons:
* We don't want to have to pass around a specific server everywhere
* We want to reuse these handler higher order methods, as then we have consistency in different usages/only one place needs to care about the API path
* If we separate the handlers, then we can reuse them in either a mock server, or a mock worker (which would let us intercept and mock the APIs in the browser as well 🙌 )